### ISSUE TITLE
Model::findMultipleByIds ignores ID filter if $arrOptions["column"] is set

### ISSUE THREAD

**Issue by danielhoering** [contao/core-bundle#1495]
Maybe it's a special problem, but it's a problem anyway. We use the Model-Class for a large number of specific and dynamic queries with some custom options. Since we started to filter "column" by predefined values in $arrOptions, there's an issue with ID-filter in **Model::findMultipleByIds**.

For a better understanding take a look at this array_merge of $arrOptions:

https://github.com/contao/core-bundle/blob/c077941468e25b4d1247f1fbbe8915e04faf23eb/src/Resources/contao/library/Contao/Model.php#L854

The problem is, when $arrOptions already have a "column"-key, the array_merge will kick out the "id IN(x,...)"-clause and so we get always the full dataset instead of only the requested IDs.

**Comment by leofeyer** [comment]
Where does `$arrOptions['column']` come from in this case and how does it look like?

**Comment by danielhoering** [comment]
It's a query we use inside our API where some Requests get filtered by specific fields. Something like this:
```php
...
$columns = array(
    'field1' => 'filterValue1',
    'field2' => 'filterValue2',
    'field3' => 'filterValue3',
    ...
);
$queryOptions = array(
    'order'  => $orderBy." ".$orderDir,
    'column' => array_keys($columns),
    'value'  => array_values($columns),
);
$collection = ModelBase::findMultipleByIds($objectIds, $queryOptions)
...
```

**Comment by aschempp** [comment]
You cannot set `column` and `value` in options, whats what you pass in to the first arguments.

**Comment by danielhoering** [comment]
Yep. That's exactly the reason for this ticket. Imho it **should** be possible, when it's allowed to set `column`and `value`separatly. There's no reason to reset `value` when both of them are manually overwritten/set. Or change how `$arrOptions` argument behave inside the Method to avoid confusion.

**Comment by aschempp** [comment]
I disagree, why don't you just use `findBy` and pass your two arrays?
